pythonforandroid/python.py

```diff
@@ -203,6 +168,33 @@ def add_flags(include_flags, link_dirs, link_libs):
             recipe = Recipe.get_recipe('openssl', self.ctx)
             add_flags(recipe.include_flags(arch),
                       recipe.link_dirs_flags(arch), recipe.link_libs_flags())
+
+        # python build system contains hardcoded zlib version which prevents
+        # the build of zlib module, here we search for android's zlib version
+        # and sets the right flags, so python can be build with android's zlib
+        info("Activating flags for android's zlib")
+        zlib_lib_path = join(self.ctx.ndk_platform, 'usr', 'lib')
+        zlib_includes = join(self.ctx.ndk_dir, 'sysroot', 'usr', 'include')
+        zlib_h = join(zlib_includes, 'zlib.h')
+        try:
+            with open(zlib_h) as fileh:
+                zlib_data = fileh.read()
+        except IOError:
+            raise BuildInterruptingException(
+                "Could not determine android's zlib version, no zlib.h in "
+                "the NDK dir includes"
```

Comment:
Minor, but I would include the zlib_h string in the error message. Maybe slightly less user friendly, but dev friendly when reading the message and seeing that the path is wrong somehow!

Reply:
Thanks @inclement,  done at https://github.com/kivy/python-for-android/pull/1722/files#diff-9836767bbcad620e7dc529862d456f9dR185